its/plugin/src/test/java/org/sonarsource/iac/PropertiesTest.java

```diff
@@ -50,4 +57,10 @@ private void checkCustomFileIdentifierForLanguage(String projectKey, String lang
       .setProperty("sonar." + language + ".file.identifier", identifier));
     assertThat(getMeasureAsInt(projectKey, "ncloc")).isEqualTo(expectedNcloc);
   }
+
+  private void checkTerraformAwsProviderVersion(String projectKey, String version, int expectedHotspots) {
+    ORCHESTRATOR.executeBuild(getSonarScanner(projectKey, BASE_DIRECTORY + "provider/", "terraform", "aws-provider")
+      .setProperty("sonar.terraform.provider.version.aws", version));
+    assertThat(getHotspotsForProject(projectKey)).hasSize(expectedHotspots);
```

Comment:
Rather than the number of issues, I think it would be better to assert the lines on which we expect issues. It would:
* Make the assertions stronger (not green when issues are raised on unexpected lines)
* Make it easier for readers to understand where we expect to raise issues.